pom.xml

```diff
@@ -139,12 +163,6 @@
         <scope>test</scope>
     </dependency>
 
-
-    <dependency>
```

Comment:
We still use the credentials for the individual templates.  Can we remove this dependency?